### PR TITLE
section 5: data anonymization

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
DRAFT stage

### CODE REVIEW

File: episodes/introduction_to_anonymization.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+elem.value
+```
+```output
+
+```
+```python
+elem.value = 'Citizen^Almoni'
+elem
+```
+In some cases, if we are dealing with a standard element, we can modify elements by keyword:
```
Comment (gcroci2) [inline episodes/introduction_to_anonymization.md]: I think that `PatientName` is an attribute, or a property of the class. I'd use the specific correct term here

**Review by gcroci2 — CHANGES_REQUESTED** [review]
Very interesting and well done lesson :)

I added very minor comments that needs to be addressed before merging, and I added a few commits mainly for further polishing the text and correct typos (https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/20/commits/d7d495eb233c2af63b142c814a487f5e2247fe7e). I also realized that my sitk episode didn't have the keypoints section at the end, so I added that as well (https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/20/commits/b4b703305ee022fe92e2484d62a13306d83f69ea).

File: episodes/introduction_to_anonymization.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
+---
+title: "Intro to Anonymization"
+teaching: 40
+exercises: 2
```
[the remark this answers is not in the document]
Comment (drcandacemakedamoore) [reply episodes/introduction_to_anonymization.md:4]: For me the exercises and code along seem quick, but I added some time to the whole lesson time to make it 50 minutes. maybe I am underestimating, but this we will see in the pilot.

File: episodes/introduction_to_anonymization.md (hunk 1 above)
Comment (drcandacemakedamoore) [reply episodes/introduction_to_anonymization.md]: Done

**Pull request merged**